Documentation/commands.md

```diff
@@ -110,7 +110,7 @@ sha512-fa1cb92dc276b0f9bedf87981e61ecde
 If you haven't trusted the creator, it will be downloaded but not verified:
 
 ```
-$ sudo ./rkt fetch coreos.com/etcd:v2.0.0
+$ rkt fetch coreos.com/etcd:v2.0.0
```

Comment:
Shouldn't this use root shell too? Technically it's possible to fetch images as a normal user if user is in `rkt` (or something) group. But I don't think we should assume that.